db/schema.rb

```diff
@@ -110,6 +110,13 @@
     t.index ["key"], name: "index_settings_on_key", unique: true
   end
 
+  create_table "topics", force: :cascade do |t|
```

Comment:
Did you re-apply the migration after the changes? Because I don't see the constraints here

Reply:
You're right, I forgot to run the migrate again